docs/fastapi-integration.md

```diff
@@ -91,6 +89,10 @@ app = FastAPI()
 async def handle_graphql_explorer(request: Request):
     return await graphql_app.handle_request(request)
 
+# Handle OPTIONS requests for CORS
+@app.options("/graphql/")
```

Comment:
You can apply multiple decorators on single view:

```python
@app.options("/graphql/")
@app.get("/graphql/")
async def handle_graphql_explorer(request: Request):
    return await graphql_app.handle_request(request)
```

Reply:
right but was thinking for the purposes of docs + the ability to put comments on each it might make more sense to split up

Reply:
I'm not saying no to comment, its good to have it. Just speaking from experience some people will copy-paste this code and then return complaining that code example is not copy-paste ready 😅